### DISCUSSION TITLE
Are there any complex scenarios?

### DISCUSSION THREAD

**Discussion by JiayunjieJYJ** [bulletphysics/bullet3#4078] [Q&A]
Are there any complex scenarios? Such as a bedroom, kitchen, or a big house? I want to train my model in complex scenarios.

Thanks a lot!

**Comment by rohit-kumar-j** [comment]
@JiayunjieJYJ try [Gibson environments](http://gibsonenv.stanford.edu/)

**Comment by JiayunjieJYJ** [comment]
@rohit-kumar-j Thanks! How can I use Gibson environments in Pybullet?
